com.unity.render-pipelines.universal/Runtime/Passes/CopyDepthPass.cs

```diff
@@ -63,7 +63,11 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             using (new ProfilingScope(cmd, ProfilingSampler.Get(URPProfileId.CopyDepth)))
             {
                 RenderTextureDescriptor descriptor = renderingData.cameraData.cameraTargetDescriptor;
+
                 int cameraSamples = descriptor.msaaSamples;
+                // When auto resolve is supported or multisampled texture is not supported, set camera samples to 1
+                if (SystemInfo.supportsMultisampleAutoResolve || SystemInfo.supportsMultisampledTextures == 0)
+                    cameraSamples = 1;
```

Comment:
Can we maybe update the comment above for default case in the switch statement below?